source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -71,7 +71,7 @@ namespace PlayFab
         return this->settings;
     }
 
-    void PlayFabEventPipeline::IntakeEvent(std::shared_ptr<const IPlayFabEmitEventRequest> request)
+    void PlayFabEventPipeline::IntakeEvent(const std::shared_ptr<const IPlayFabEmitEventRequest>& request)
```

Comment:
disagree

Reply:
Since "IntakeEvent" sounds like it would imply transfering ownership, sure, we can ignore this change too.